src/views/layouts/shell.njk

```diff
@@ -14,7 +14,7 @@
                 : 'js-enabled');
         </script>
         {% include "partials/cookie_consent_banner.njk" %}
-        <div role="navigation" aria-label="skip to main content">
+        <div role="navigation" aria-label="lang.skip_to_main_content">
             <a href="#main-page-content" class="govuk-skip-link">Skip to main content</a>
```

Comment:
Does this work okay without brackets like you have in default.njk?

Reply:
Well spotted. It's fixed now.